src/schema/organization/contributors/resolvers.js

```diff
@@ -1,61 +1,131 @@
 import fetchData from '../../../github/dataFetcher';
 import githubQueries from '../../../github/queries';
-import { GeneralSet, getOldestDate } from '../../../utils'
+import { getOldestDate } from '../../../utils';
+
+/**
+ * @class A variant of Set, which the difference is on the equality condition of objects
+ */
+export class ContributorSet {
```

Comment:
Why you don't use native `Set`?

Reply:
I think that the answer is on the documentation above: `A variant of Set, which the difference is on the equality condition of objects`.

Reply:
Exactly as @LukeHxH said.